api/geth_backend.go

```diff
@@ -186,7 +186,7 @@ func (b *GethStatusBackend) ensureAppDBOpened(account multiaccounts.Account, pas
 	return nil
 }
 
-func (b *GethStatusBackend) SaveAccountAndStartNodeWithKey(acc multiaccounts.Account, conf *params.NodeConfig, password string, keyHex string) error {
+func (b *GethStatusBackend) SaveAccountAndStartNodeWithKey(acc multiaccounts.Account, conf *params.NodeConfig, password string, subaccs []accounts.Account, keyHex string) error {
```

Comment:
```suggestion
func (b *GethStatusBackend) SaveAccountAndStartNodeWithKey(conf *params.NodeConfig, acc multiaccounts.Account, password string, keyHex string, subaccs []accounts.Account) error {
```

Would this order make more sense?

Reply:
I switched password and config so that it has the same params order than StartNodeWithAccountAndConfig